### PR TITLE
MM 31935 - fill issue detail in contact-us form

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
Contact Us Form - pre-fill the issue details in the form when the value is sent as a URL parameter

System Console > Billing & Account > Subscription > [Bottom of the page] Cancel your subscription section > Click -Contact us- button

This PR adds the prefill functionality to the inquiry issue as shown in the attached image

#### Ticket Link
[MM-31935](https://mattermost.atlassian.net/browse/MM-31935)
#### Related Pull Requests
- Has customer web portal Changes => [PR310](https://github.com/mattermost/customer-web-server/pull/310)

```release-note
NONE
```

#### Screenshots
Captura de pantalla 2021-01-14 a las 12 14 40

### CODE REVIEW

File: components/admin_console/billing/billing_subscriptions.tsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ const BillingSubscriptions: React.FC<Props> = () => {
     const preferences = useSelector<GlobalState, PreferenceType[]>((state) => getCategory(state, Preferences.ADMIN_CLOUD_UPGRADE_PANEL));
 
     const contactSalesLink = useSelector((state: GlobalState) => getCloudContactUsLink(state, InquiryType.Sales));
+    const contactUsLink = useSelector((state: GlobalState) => getCloudContactUsLink(state, InquiryType.Sales, InquiryIssue.CancelAccount));
```
Comment (marianunez) [inline components/admin_console/billing/billing_subscriptions.tsx:63]: Given that this is a specific cancel account link now, I suggest: 
```suggestion
    const cancelAccountLink = useSelector((state: GlobalState) => getCloudContactUsLink(state, InquiryType.Sales, InquiryIssue.CancelAccount));
```

**Review by marianunez — APPROVED** [review]
LGTM, thanks @pablovelezvidal

**Review by stevemudie — APPROVED** [review]
Looks good. Will test post merge.

**Review by nickmisasi — APPROVED** [review]
LGTM!

**Pull request merged**